.github/CODEOWNERS

```diff
@@ -2027,19 +2027,46 @@ x-pack/test/profiling_api_integration @elastic/obs-ux-infra_services-team
 x-pack/plugins/observability_solution/observability_shared/public/components/profiling @elastic/obs-ux-infra_services-team
 
 # Shared UX
-/test/api_integration/apis/guided_onboarding @elastic/appex-sharedux
+/test/interpreter_functional @elastic/appex-sharedux # Assigned per https://github.com/elastic/kibana/pull/199767/files/9eb7f6e32e7645ffde6e5e8eaae72bf7f15a6f70#r1838041461
+/test/accessibility/apps/home.ts @elastic/appex-sharedux # Assigned per https://github.com/elastic/kibana/pull/199771/files#r1840077237
+/test/api_integration/apis/home/*.ts @elastic/appex-sharedux # Assigned per https://github.com/elastic/kibana/pull/199771/files#r1840077065
+/test/functional/apps/home @elastic/appex-sharedux # Assigned per https://github.com/elastic/kibana/pull/199771/files#r1840075278
+/x-pack/test/plugin_functional/plugins/global_search_test @elastic/appex-sharedux
+/test/functional/services/saved_objects_finder.ts @elastic/appex-sharedux
+/test/interpreter_functional/test_suites/run_pipeline @elastic/appex-sharedux
+/test/functional/apps/kibana_overview @elastic/appex-sharedux
+/x-pack/test/functional_solution_sidenav/config.ts @elastic/appex-sharedux
+/x-pack/test/functional_solution_sidenav/ftr_provider_context.ts @elastic/appex-sharedux
+/x-pack/test/functional_solution_sidenav/services.ts @elastic/appex-sharedux
+/x-pack/test/functional_solution_sidenav/index.ts @elastic/appex-sharedux
+/x-pack/test/functional_cloud @elastic/appex-sharedux
+/x-pack/test/functional/services/user_menu.js @elastic/appex-sharedux # Assigned per https://github.com/elastic/kibana/pull/179270
+/x-pack/test/functional/services/ace_editor.js @elastic/appex-sharedux
```

Comment:
This service can be removed. Fine to keep it the assignment in the PR, will follow up later

Reply:
Actually... it seems that the service is still used by test, which surprises me because I thought we completely removed ace editor. I think this better be assigned to @elastic/kibana-management

Reply:
Send this our way, seems I might have forgotten to remove these specific test service. I'll create an issue to address it. Odd that tests that depend on it still pass since the ace dependency is gone 🥲

Reply:
@sabarasaba It's been sent your way! :)